typing/includecore.ml

```diff
@@ -545,7 +548,7 @@ module Variant_diffing = struct
               cd1.cd_res cd2.cd_res cd1.cd_args cd2.cd_args with
       | Some reason ->
           Error (Diffing_with_keys.Type {pos; got=cd1; expected=cd2; reason})
-      | None -> Ok (Ident.name cd1.cd_id)
+      | None -> Ok ()
```

Comment:
I suspect that this was a leftover from a previous implementation approach, that was accepted for no good reason due to lax typing.

Reply:
Most probably.